rules/code-quality/tests/Rector/FuncCall/ArrayKeysAndInArrayToArrayKeyExistsRector/ArrayKeysAndInArrayToArrayKeyExistsRectorTest.php

```diff
@@ -6,16 +6,16 @@
 
 use Iterator;
 use Rector\CodeQuality\Rector\FuncCall\ArrayKeysAndInArrayToArrayKeyExistsRector;
-use Rector\Core\Testing\PHPUnit\AbstractRectorTestCase;
 
-final class ArrayKeysAndInArrayToArrayKeyExistsRectorTest extends AbstractRectorTestCase
+final class ArrayKeysAndInArrayToArrayKeyExistsRectorTest extends RunnableTestCase
 {
     /**
      * @dataProvider provideData()
      */
     public function test(string $file): void
     {
         $this->doTestFile($file);
+        $this->assertOriginalAndFixedFileYieldSameResult($file);
```

Comment:
yield is PHP keyword that reminds something related to collections

What about assert...ResultEquals?

Reply:
makes sense